services/core-web/src/setupTests.js

```diff
@@ -18,6 +18,12 @@ jest.mock("react-lottie", () => ({
   default: "lottie-mock",
 }));
 
+jest.mock("@mds/common/providers/featureFlags/useFeatureFlag", () => ({
```

Comment:
Weird.  I have a memory of adding this to setupTests, but I mustn't have committed it for some reason.  Nice!

Reply:
I think it was on MS but not on Core! You probably remember it right!